src/main/resources/application.properties

```
@@ -0,0 +1,8 @@
+spring.application.name=school
+spring.datasource.url= jdbc:postgresql://localhost:5432/hogwarts
+spring.datasource.username= student
+spring.datasource..password= chocolatefrog
+spring.datasource.driver-class-name=org.postgresql.Driver
+
+
+spring.jpa.hibernate.ddl = update
```

Неправильное имя свойства

```suggestion
spring.jpa.hibernate.ddl-auto = update
```